src/views/shared/SecurityBeacon.js

```diff
@@ -120,7 +140,8 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         <div class="circle right"></div>\
       </div>\
     </div>\
-    <div class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+    <div aria-live="polite" role="alert" class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
```

Comment:
After thinking it over, I am going with role="image" because that is what the customer asked for. It doesn't read out loud in Voiceover for me but as I said earlier, I would rather not get ahead of them.